src/main/scala/is/hail/sparkextras/OrderedRDD2.scala

```diff
@@ -699,6 +710,8 @@ class OrderedRDD2 private(
       rdd2) {
       override def getStorageLevel2: StorageLevel = level
 
+      override def isPersisted: Boolean = sparkContext.getPersistentRDDs.contains(persistedRDD.id)
```

Comment:
I see my code is slightly wrong.  I think it is better to directly check the storage level here because this is somewhat ambiguous.  (persisted to what level?)  However, that might not work because my code here was broken.  Instead of checking `level`, we should be checking persistedRDD.storageLevel.  That, or we have to update level = StorageLevel.NONE when we unpersist.